src/gui/folder.cpp

```diff
@@ -95,6 +95,9 @@ Folder::Folder(const FolderDefinition &definition,
 
     _syncResult.setFolder(_definition.alias);
 
+    // those errors should not persist over sessions
+    _journal.wipeErrorBlacklistCategory(SyncJournalErrorBlacklistRecord::Category::LocalSoftError);
```

Comment:
Would somehow be cool if we had a logging if there were blacklisted errors from the previous run. That might be an important hint when debugging. No idea how difficult that is...

Reply:
Hm true, in general `wipeErrorBlacklist*` should log what was wiped